dictionaries/html/html.txt

```diff
@@ -1,21 +1,84 @@
 a
+aacute
+Aacute
```

Comment:
I left both casings for the entites, since some are different in that they are the actual upper and lower case letter